OneCommunityGlobal/HighestGoodNetworkApp
# Jatin get profile images from website

# Description
<img width="733" alt="Screenshot 2024-09-08 at 1 41 52 PM" src="https://github.com/user-attachments/assets/8d53733f-5842-4077-8503-073a5fcc413d">

## Related PRS (if any):
This frontend PR is related to the [#1116](https://github.com/OneCommunityGlobal/HGNRest/pull/1116/) backend PR.

## Main changes explained:
- Added `confirmRemoveModal.js` to `components/UserProfileModal`.
- Added `suggestedProfileModal.js` to `components/UserProfileModal`.
- Made Changes to `UserProfileModal.css` and `UserProfile.jsx` files.
…

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as any user
5. go to UserProfile Page.
6. If you have a profile on the onecommunity website, you should either have a profile image(if it was empty before) else you should see a `suggested Profile Image` button to display all the images that match your name.

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/b60da6c0-0375-4fc8-9a5a-e2912c29463f
